src/core/AdminGuesser.tsx

```diff
@@ -95,6 +96,10 @@ const AdminGuesser = ({
     [setLoading, setIntrospect],
   );
 
+  if (loading) {
+    return React.createElement(Loading, null);
```

Comment:
Maybe could we use JSX?

```suggestion
    return <Loading />;
```